src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -252,6 +270,7 @@ def execute(self, reqStatus):
         self.updateReportDict(summary, "num_datasets_subscribed", self.dsetCounter)
         self.updateReportDict(summary, "num_blocks_subscribed", self.blockCounter)
         self.updateReportDict(summary, "nodes_out_of_space", list(self.rseQuotas.getOutOfSpaceRSEs()))
+        self.updateReportDict(summary, "site_list_update_errors", siteUpdateErrorCount)
```

Comment:
I can not find in the code where the variable `siteUpdateErrorCount` is incremented. did i miss anything or it was maybe removed in a previous commit?

Reply:
good catch, it came from previous revisions of the development cycle.

Reply:
good, thanks!